config/config.py.template

```diff
@@ -455,7 +505,7 @@ NAIVE_BAYES_TEST_OPTS = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
     # Naive Bayes smoothing lambda.
     OptionSet("nb-lambda", [1.0]),
     # Model type: either Multinomial or Bernoulli
-    OptionSet("model-type", ["Multinomial"]),    
```

Comment:
While you're at it, can you revert the change on this line since it will cause a conflict with #85